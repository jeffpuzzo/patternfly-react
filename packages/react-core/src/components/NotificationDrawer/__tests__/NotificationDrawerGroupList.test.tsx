import React from 'react';
import { render, screen } from '@testing-library/react';
import { NotificationDrawerGroupList } from '../NotificationDrawerGroupList';

describe('NotificationDrawerGroupList', () => {
  test('renders with PatternFly Core styles', () => {
    render(<NotificationDrawerGroupList data-testid="test-id" />);
    expect(screen.getByTestId('test-id').outerHTML).toMatchSnapshot();
  });

  test('className is added to the root element', () => {
    render(<NotificationDrawerGroupList className="extra-class" data-testid="test-id" />);
    expect(screen.getByTestId('test-id').className).toMatchSnapshot();
  });
});
