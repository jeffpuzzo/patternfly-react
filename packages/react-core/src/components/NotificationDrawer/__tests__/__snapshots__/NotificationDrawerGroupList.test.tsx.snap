// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationDrawerGroupList className is added to the root element 1`] = `"pf-c-notification-drawer__group-list extra-class"`;

exports[`NotificationDrawerGroupList renders with PatternFly Core styles 1`] = `"<div data-testid=\\"test-id\\" class=\\"pf-c-notification-drawer__group-list\\"></div>"`;
